python/tests/test_twitter.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import sys, os
+sys.path.append(os.path.abspath(os.path.join('..', 'fastquant')))
+
+from fastquant.data import *
+
+def test_get_twitter_sentiment():
+  # You need to put your own Twitter API Credentials here
+  consumer_key = ''
```

Comment:
This means the test will not work since the credentials are blank right? I'm okay commenting out the part that requires creds for now and just do a unit test for `tweepy_api`.

Reply:
Hi @enzoampil, ah yes got it. Sorry, I've overlooked that portion. I'll just send a new PR for the unit test for tweepy_api, but I'm not sure how to input the API credentials without displaying it there. If ever, do I just add the function test in the same file as test_twitter.py?

Thank you also @enzoampil , @rafmacalaba , @jpdeleon for the guidance!